### PR TITLE
Import Hardhat config from toolkit

### PR DESCRIPTION
**Pull request by fadeev**
Proposed default HH config:

```ts
import "@nomicfoundation/hardhat-toolbox";
import "@zetachain/toolkit/tasks";
import * as dotenv from "dotenv";

import { getHardhatConfig } from "@zetachain/toolkit/client";
import { HardhatUserConfig } from "hardhat/config";

dotenv.config();

export const config: HardhatUserConfig = {
  ...getHardhatConfig({ accounts: [process.env.PRIVATE_KEY] }),
};
```

I thought about initializing a client as part of HH config, but then it becomes cumbersome to pass the client to HH tasks.

### CODE REVIEW

**Review by andresaiello — APPROVED** [review]

**Comment by fadeev** [comment]
@lukema95 can you, please, take a look at this PR? Thanks!

File: packages/client/src/index.ts [hunk 1]
```diff
@@ -5,6 +5,8 @@ export * from "./getChainId";
 export * from "./getEndpoint";
 export * from "./getFees";
 export * from "./getForeignCoins";
+export * from "./getHardhatConfig";
+export * from "./getHardhatConfig";
```
Comment (lukema95) [inline packages/client/src/index.ts:9]: It looks like lines 8 and 9 duplicate each other.

**Comment by lukema95** [comment]
> @lukema95 can you, please, take a look at this PR? Thanks!

Everything looks good to me, I only noticed a minor problem with two lines of code redundancy.

File: packages/client/src/index.ts (hunk 1 above)
Comment (fadeev) [reply packages/client/src/index.ts]: 59aa44c

**Pull request merged**